File: 백준/Bronze/1712. 손익분기점/손익분기점.java
import java.io.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));

        String[] input = br.readLine().trim().split(" ");
        int A = Integer.parseInt(input[0]);
        int B = Integer.parseInt(input[1]);
        int C = Integer.parseInt(input[2]);
        int result = 0;
        if(B >= C) result = -1;
        else result = A / (C- B) + 1;

        bw.write(String.valueOf(result));
        bw.flush();
        br.close();
        bw.close();
    }
}

File: 백준/Bronze/1712. 손익분기점/손익분기점.py
import sys

input = sys.stdin.readline

A, B, C = map(int, input().strip().split())

result = 0

if C - B <= 0:
    result = -1
else:
    result = A // (C - B) + 1

sys.stdout.write(str(result) + "\n")